### DISCUSSION TITLE
Can we manipulate the fanflat (cone) beam  angle for 2d (3d) reconstruction?

### DISCUSSION THREAD

**Discussion by figueroa1395** [astra-toolbox/astra-toolbox#412] [Q&A] [answered]
I can't seem to find what is the default fanflat (cone) beam angle used in the toolbox for 2d (3d) reconstruction. Could you tell me where I can find it?

Additionally, is it possible to manipulate this angle at will before each reconstruction? If so, how can we do this? This is very important in order to replicate the parameters fixed in experimental setups and also to determine optimal parameters such as the distances from source to target, target to detectors, etc.

Thank you in advance!

**Comment by askorikov** [comment, marked as answer]
Hi! The fanflat (and similarly cone) angle is not specified directly, but is instead controlled by other parameters in the following way: `2 * atan(0.5 * det_spacing * det_pixel_count / (source_origin_distance + origin_detector_distance))`. Let me know if it makes sense.

**Reply by figueroa1395** [reply]
It makes perfect sense. This solves my question. Thank you.
